Core/include/Acts/EventData/SingleFreeTrackParameters.hpp

```diff
@@ -89,10 +87,10 @@ class SingleFreeTrackParameters {
   /// be retrieved
   ///
   /// @return Value of the requested parameter uncertainty
-  template <unsigned int par,
+  template <FreeParametersIndices par,
```

Comment:
Naming: to follow our naming convention `par` should be something like `kIndex` or `kIdx`.

Reply:
Done